tests/Feature/Subscription.php

```diff
@@ -748,6 +748,7 @@ public function testFindSubscriptionsByCustomerIdWithStatusFilter(callable $subs
         $this->assertCount(2, $subscriptions);
         $this->assertEquals(Status::pending(), $subscriptions[0]->getStatus());
         $this->assertEquals(Status::active(), $subscriptions[1]->getStatus());
+        $this->assertNotNull($customer->getDefaultPaymentMethod()->getExpirationDate());
```

Comment:
@johncorrelli do you see any issue with relying on `$subscription->getCustomer()->getDefaultPaymentMethod()` for our current need?

Braintree subscription results do not contain things like credit card info on the root of the response. It is only available on transactions within the subscription.

Reply:
Do we have the ability to get all `paymentMethods` for a customer? It's not guaranteed that the subscription will be using the default payment method. It *should* be in our case, but it's not guaranteed in all cases.

Although, this is in the context of a test, so it's probably fine here, since that should be a fresh customer anyways.

But how we implement this on within the teamgantt api might need to be something like this...

```php
$subscriptionPaymentTokenId = "foobar";
$paymentMethod = array_filter($subscription->getCustomer()->getPaymentMethods(), fn ($method) => $method->getId() === $subscriptionPaymentTokenId);
```

_not 100% sure the function names / payment token names are :100: but you get the idea_